example/distributed_training-horovod/mxnet_imagenet_resnet50.py

```diff
@@ -0,0 +1,456 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
copyright

Reply:
do you just copy the example from Horovod here?

Reply:
Thanks. Updated.